charts/navi-back/templates/configmap.yaml

```
@@ -29,16 +29,11 @@ data:
       {{- /* distribute equally fot HTTP/GRPC */}}
       {{- $httpHandlers := 0 }}
       {{- $grpcHandlers := 0 }}
-      {{- if .Values.naviback.grpcPort }}
-          {{- $httpHandlers = div .Values.naviback.handlersNumber 2 }}
-          {{- $grpcHandlers = sub .Values.naviback.handlersNumber $httpHandlers }}
-      {{- else }}
-          {{- $httpHandlers = .Values.naviback.handlersNumber }}
-      {{- end }}
       "http": {
         "server": "0.0.0.0",
         "port": {{ .Values.naviback.appPort }},
-        "handlers_number": {{ $httpHandlers }},
+        "handlers_number": {{ .Values.naviback.handlersNumber | int }},
+        "queue_size": {{ .Values.naviback.queueSize }},
```

А почему здесь нет функции int, как выше?
```suggestion
        "queue_size": {{ .Values.naviback.queueSize | int }},
```

---

добавил